static/src/javascripts/bootstraps/digitalpack.js

```diff
@@ -0,0 +1,3 @@
+define(['mmaTabs/digitalpack-tab'], function (digipackTab) {
```

Comment:
Why are these necessary? Can't we just require `mmaTabs/digitalpack-tab` directly in `account-profile.js`?